openspec/changes/project-runtime-01-safe-artifact-write-policy/specs/backlog-add/spec.md

```
@@ -0,0 +1,9 @@
+## ADDED Requirements
+
+### Requirement: Backlog add local artifact helpers SHALL preserve user-managed content
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)